os/board/rtl8730e/Kconfig

```diff
@@ -20,6 +20,17 @@ config FLASH_START_ADDR
         A start address of flash (in hex).
         This is fixed value, so user doesn't need to change it.
 
+config TRPK_CONTAINS_MULTIPLE_BINARY
+    bool
+    default n
+    depends on XIP_ELF
+    help
+        This macro is to differentiate whether the kernel trpk contains
+        more than 1 binary. If yes, we have to recalculate the offset
+        according to the binary that is running on Tizen Lite, to make
+        sure the loadable_apps for XIP_ELF is placed to the correct
+        virtual flash address. 
```

Comment:
The description is the same with the flat. But we don't need this for the flat. Are you able to leave why `XIP_ELF` requires this more?

Reply:
Hi @sunghan-chang , could you check the comments again?